src/styles/blocks/calendar.scss

```
@@ -0,0 +1,41 @@
+.calendar {
+
+  width: $calendar-width;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+
+  @each $montName, $orderNumber in $startDay {
```

```suggestion
  @each $dayName, $orderNumber in $startDay {
```
Why it's montName, if it's actually dayName of day of the week?